Northstar.Client/mod/scripts/vscripts/ui/menu_lobby.nut

```diff
@@ -1627,4 +1627,4 @@ void function Lobby_SetFDModeBasedOnSearching( string playlistToSearch )
 	}
 
 	Lobby_SetFDMode( isFDMode )
-}
\ No newline at end of file
+}
```

Comment:
Unrelated file change (added newline). Maybe revert to make the diff cleaner? ^^

Reply:
that must have been github doing that automatically when i introduced a compile error via web editor, will revert